prow/config.yaml

```diff
@@ -137,9 +137,9 @@ presubmits:
     - name: pull-kubernetes-e2e-kubeadm-gce
       agent: kubernetes
       context: pull-kubernetes-e2e-kubeadm-gce
-      always_run: false
       max_concurrency: 8
       skip_report: true
+      run_if_changed: '^cmd\/kubeadm\/.*\.go$'
```

Comment:
shouldn't this trigger if _any_ file in kubeadm changes, eg BUILD?

Reply:
I think .go is very fine. I don't see a case where a BUILD file would change but not a .go file that would be important to catch for this job.
There are only go and BUILD files in this dir, so this is ok

Reply:
BUILD can change when the buildfile generation is changed.

Reply:
Since there are only go and BUILD there's not really any reason to filter the file type, but as-is this wouldn't be built if the BUILD files were updated without changing the source, which definitely happens.

Reply:
neat... I was not aware of this feature!

Reply:
You don't really need to escape the slashes. Also, I agree with @BenTheElder, keeping `cmd/kubeadm` only in the regexp should be enough.